packages/design-system/src/molecules/alertV2/index.tsx

```diff
@@ -0,0 +1,98 @@
+import { View, StyleSheet } from 'react-native'
+import { TypographyV2, Icon } from '../../atoms'
+import { useThemeV2 } from '../../hooks'
+import { ButtonV2 } from '../../atoms'
+import { useBreakpointValue } from 'native-base'
+
+type AlertProps = {
+  title: string
+  variant?: 'info' | 'danger'
+  showButton?: boolean
+  buttonText?: string
+  textColor?: string
+}
+
+const AlertV2: React.FC<AlertProps> = ({
+  title,
+  variant = 'info',
+  showButton = false,
+}) => {
+  const theme = useThemeV2()
+
+  const isDanger = variant === 'danger'
+  const backgroundColor = isDanger
+    ? theme.base.danger
+    : theme.base.highlight2[10]
+  const iconColor = isDanger ? theme.text.white[100] : theme.text.black[100]
+  const titleColor = isDanger ? theme.text.white[100] : theme.text.black[100]
+  const iconBackgroundColor = isDanger ? theme.base.primary[10] : ''
+
+  const styles = StyleSheet.create({
+    container: {
+      borderRadius: 16,
+      padding: 16,
+      margin: 16,
+      display: 'flex',
+      flexDirection: 'row',
+      alignItems: 'center',
+      //   alignSelf: 'flex-start',
+      maxWidth: useBreakpointValue({ base: '100%', md: 600 }),
+      alignSelf: useBreakpointValue({ base: 'stretch', md: 'flex-start' }),
+      backgroundColor,
+    },
+    iconContainer: {
+      marginRight: 8,
+      backgroundColor: iconBackgroundColor,
+      padding: 6,
+      borderRadius: 100,
+    },
+    titleText: {
+      titleColor,
+      fontSize: 16,
+    },
+    textContainer: {
+        flex: 1,
+        padding: 8,
+    },
+    buttonContainer: {
+      marginLeft: useBreakpointValue({ md: 16 }),
```

Comment:
You should not create styles inside the component. If you need to use a hook in the `StyleSheet.create`, you have to create a new hook called `useStyles` & use like this:

```tsx
const useStyles = () => {
  const buttonContainerMarginLeft = useBreakpointValue({ md: 16 })
  const styles = useMemo(() => StyleSheet.create({}), [buttonContainerMarginLeft])

  return styles
}
```

Reply:
If you don't memoize, the styles object will be a new instance on each component render.

Reply:
Yeah! you can also use inline styles for this kind of values, it will look something like this:
```tsx
const buttonContainerMarginLeft = useBreakpointValue({ md: 16 }) 
return (
  <View styles={[styles.buttonContainer, { marginLeft: buttonContainerMarginLeft }]} />
)

```